package-lock.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "ansible-language-server",
+  "name": "@ansible/ansible-language-server",
   "version": "0.1.1-0",
```

Comment:
Should the version be started. from 0.0.1 as it is a new package?

Reply:
This is the result of running `npm ci`, the change was not manually created. We missed to include it in #10 due to lack of CI setup on this repository.

Reply:
We can probably update the version here https://github.com/ansible/ansible-language-server/blob/main/package.json#L7 
Don't see a reason to have version referenced from a older deprecated package.

Reply:
/cc @webknjaz

Reply:
IMHO, updating the version is outside the scope of this PR. We will increase it for sure but just before publishing it, after we get CI pipelines setup.

I already know how to prevent this mistake in the future as we implemented such measure in vscode-ansible a day or two ago.

Reply:
Agreed it is outside the scope of the PR but thought of commenting it here since I missed reviewing the PR that caused this change.

Reply:
@ganeshrn let's make sure to talk about the version bump before making the release. I'm fine with continuing the version numbers. Because technically it's still the same project. Could you mark this conversation as resolved and create an issue if there's anything further to discuss?